docs/pages/es/functionality/performance.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Performance Management
+permalink: es/functionality/performance-management
+type: functionality
+---
+
+# Gestión del rendimiento
+La clave del funcionamiento eficiente de cualuier service mesh es la medición y gestión de su rendimiento.

```

Comment:
In the next sentences, some letter is missing. These words are in italics: _cualquiera_